### PR TITLE
Remove ghost from the codebase

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Now that `ws-scheduler` was removed and we are not using ghost workplaces anymore, remove it

## Release Notes

```release-note
Remove ghost from the codebase
```

### CODE REVIEW

File: components/ws-manager-api/typescript/src/core_pb.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
```
Comment (mads-hartmann) [inline components/ws-manager-api/typescript/src/core_pb.js:2]: I was under the impression we shouldn't update these ([ref](https://gitpod.slack.com/archives/C01KGM9AW4W/p1645090427983519))

File: components/ws-manager-bridge/src/bridge.ts.rej [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- components/ws-manager-bridge/src/bridge.ts
```
Comment (mads-hartmann) [inline components/ws-manager-bridge/src/bridge.ts.rej:1]: Are the .orig and .rej added on purpose?

**Review by mads-hartmann — COMMENTED** [review]
Awesome 🎉 I left two clarifying comments that I'd like to have addressed before I can approve ☺️

File: components/ws-manager-api/typescript/src/core_pb.js (hunk 1 above)
Comment (geropl) [reply components/ws-manager-api/typescript/src/core_pb.js:2]: :100: :point_up:

**Comment by princerachit** [comment]
Everything looks good, would like to get info on @mads-hartmann 's question and did we test this with helm ?

File: components/ws-manager-bridge/src/bridge.ts.rej (hunk 2 above)
Comment (geropl) [reply components/ws-manager-bridge/src/bridge.ts.rej:1]: Those seem to be merge left-overs. Please remove, the rest LGTM!

Comment (aledbf) [reply components/ws-manager-bridge/src/bridge.ts.rej]: Removed

**Review by geropl — APPROVED** [review]
LGTM

**Review by mads-hartmann — DISMISSED** [review]
LGTM 🚀  The only outstanding comment is about the copyright header update, but sounds like it's an autogenerated so might not be relevant ([comment reference](https://github.com/gitpod-io/gitpod/pull/8363/files#r811675399))

**Review by mads-hartmann — APPROVED** [review]
Can't quite figure out what the diff is between now and the last review I gave, but scanning the code things still LGTM, so approving again.

**Review by MrSimonEmms — APPROVED** [review]

**Comment by kylos101** [comment]
@aledbf can we also remove from [here](https://github.com/gitpod-io/gitpod/blob/d9b9f3087b2570a481e800d04846fbafe53087f9/install/installer/pkg/config/v1/config.go#L223)?

**Review by geropl — APPROVED** [review]

**Review by kylos101 — APPROVED** [review]
LGTM

**Comment by mustard-mh** [comment]
Should we clean these up too?

image

**Review by mustard-mh — APPROVED** [review]

**Review by csweichel — APPROVED** [review]

**Pull request merged**